cura/CuraApplication.py

```diff
@@ -1151,7 +1151,7 @@ def reloadAll(self):
             if file_name:
                 job = ReadMeshJob(file_name)
                 job._node = node
-                job.finished.connect(self._reloadMeshFinished)
+                job.finished.connect(node._reloadJobFinished)
```

Comment:
`SceneNode` has no function `_reloadJobFinished`. Pressing F5 now crashes Cura.

Reply:
There was a PR months before. There I introduced or just renamed this signal.